libbeat/publisher/publish.go

```diff
@@ -87,6 +87,14 @@ type BeatPublisher struct {
 	numClients uint32
 }
 
+type BeatInfo struct {
```

Comment:
Will move it to `common` for now. Seems to be the place to prevent potential circular dependencies for now.